core/web/resolver/eth_transaction.go

```diff
@@ -73,7 +73,7 @@ func (r *EthTransactionResolver) Nonce() *string {
 		return nil
 	}
 
-	value := stringutils.FromInt64(*r.tx.Sequence)
+	value := stringutils.FromInt64((*r.tx.Sequence).Int64())
```

Comment:
Isn't `Sequence` a `fmt.Stringer`?
```suggestion
	value := r.tx.Sequence.String()
```

Reply:
+1